buildmimic/bigquery/README.md

```diff
@@ -0,0 +1,221 @@
+2. https://mimic.physionet.org/about/mimic/
+3. https://mit-lcp.github.io/mimic-schema-spy/
+
+We didn't set any column as REQUIRED (i.e., NULL values are not allowed) because the tables will be used for searching/querying only and users with Viewer role should not be able to modify the tables (ex: insert/delete/update data). However, depending on your environment, you may need to set the mode REQUIRED for the appropriate columns. 
+
+We selected the type DATAETIME for all columns with dates and times. On the MIMIC webpage the data types informed represent the PostgreSQL data types. Date and time, for example, are listed as TIMESTAMP(0). PostgreSQL doesn't seem to have a type DATETIME, but BigQuery does and that may be a more appropriate representation. In addition, the TIMESTAMP type was adding digits for second precision (which MIMIC doesn’t have) and setting UTC timezone as default. Columns with suffix DATE (ex: CHARTDATE) “will always have 00:00:00 as the hour, minute, and second values. This does not mean it was recorded at midnight: it indicates that we do not have the exact time, only the date” (source: https://mimic.physionet.org/mimicdata/time). This is also true for other columns such DOB and DOD (Patients table).
+
+---
+## STEP 6: Create tables and load the csv files
+
+### A) Create a script file (ex: upload_MIMIC3_v1_4.sh) and copy the code below. Change the **schema_local_folder** to match the path to the schemas on your local machine.
+
+```
+#!/bin/bash
+
+# Initialize parameters
+bucket="mimic3_v1_4"
+dataset="MIMIC3_V1_4"
+schema_local_folder="/home/user/mimic3_schema"
+
+# Get the list of files in the bucket
+FILES=$(gsutil ls gs://$bucket)
+
+for file in $FILES
+do
+
+# Extract the table name from the file path (ex: gs://mimic3_v1_4/ADMISSIONS.csv)
+base=${file##*/}       # remove path
+tablename=${base%.*}   # remove extension .csv
+
+# Create table and populate it with data from the bucket
+bq load --allow_quoted_newlines --skip_leading_rows=1 --source_format=CSV $dataset.$tablename gs://$bucket/$tablename.csv $schema_local_folder/$tablename.schema.json
+
+# Check for error
+if [ $? -eq 0 ];then
+    echo "OK....$tablename"
+else 
+    echo "FAIL..$tablename"
+fi
+
+done
+exit 0
+```
+
+This code will get the list of files in the bucket, and for each file, it will extract the table name (ex: ADMISSIONS). With the table name, the system executes the BigQuery load command `bq load` to create a table and load the data from the csv file from the bucket using the specific schema from the local folder. The script will need the name of the bucket where the csv files are stored on Cloud Storage, the name of the dataset to create and upload the tables, and the path to the JSON file (schema) on your local machine.
+
+### B) Set the CHMOD to allow the file as executable (ex: 755), and execute the script file.
+
+```
+./upload_MIMIC3_v1_4.sh
+```
+
+During the load with the use of schemas the following error occurred: 
```

Comment:
Since the above script already includes --allow_quoted_newlines, maybe put this documentation up in section A? In this context it might confuse the user into thinking they need to change something.